### DISCUSSION TITLE
"No SED found. No plot created."

### DISCUSSION THREAD

**Discussion by nymeriag** [mboquien/cigale#69] [Q&A] [answered]
Hello! I'm currently working on SED fitting using CIGALE for a Type-2 Seyfert Galaxy with a redshift of 0.002642. Despite adjusting my configuration and attempting various parameters, I'm encountering a persistent error: "No SED found/No plot created." I've also tried running the configurations shared online, but the error persists. I'm wondering if the issue might be related to my CIGALE installation. So, I reinstalled, but still yields the same error. Could you kindly review the parameters I've gathered and suggest any additional ones I should consider? Also, are there any prerequisites I might be missing before initiating the SED fitting process? Hoping for your help! Thank you!

[photometric_data.csv](https://github.com/mboquien/cigale/files/13051411/photometric_data.csv)
[pcigale.txt](https://github.com/mboquien/cigale/files/13051412/pcigale.txt)

**Comment by mboquien** [comment, marked as answer]
Hello. The issue is that only 1 model is computed. CIGALE computes models on a grid defined by the Cartesian product of all the parameters, so you need to provide more than a single value for the parameters you want to set free. For a single object, it is not uncommon to have a million models or more. I hope this helps.

**Reply by nymeriag** [reply]
It worked! Thank you so much for the help!
